snap/local/files/run_nv_hostengine.sh

```diff
@@ -11,4 +11,11 @@ if [ -n "$nv_hostengine_port" ]; then
     args+=("-p" "$nv_hostengine_port")
 fi
 
+# Check if dcgm-exporter service is running and restart it
+# systemctl will not work in strict confinement
+dcgm_exporter_status=$(snapctl services dcgm.dcgm-exporter | awk '/dcgm\.dcgm-exporter/ {print $3}')
+if [ "$dcgm_exporter_status" == "active" ]; then
+    snapctl restart dcgm.dcgm-exporter
+fi
```

Comment:
This was originally set to run after starting nv-hostengine; is there any concern about starting it before starting nv-hostengine?

Reply:
Not that I've seen

Reply:
Thanks :+1: